internal/component/bosh_io_index.yml

```diff
@@ -0,0 +1,390 @@
+  url: https://github.com/cloudfoundry/nats-release
+- name: postgres
+  url: https://github.com/cloudfoundry/postgres-release
+- name: cf-smoke-tests
+  url: https://github.com/cloudfoundry/cf-smoke-tests-release
+- name: statsd-injector
+  url: https://github.com/cloudfoundry/statsd-injector-release
+- name: bits-service
+  url: https://github.com/cloudfoundry-incubator/bits-service-release
+- name: cf-app-sd
+  url: https://github.com/cloudfoundry/cf-app-sd-release
+- name: silk
+  url: https://github.com/cloudfoundry/silk-release
+- name: cf-cli
+  url: https://github.com/bosh-packages/cf-cli-release
+- name: eirini
+  url: https://github.com/cloudfoundry-community/eirini-bosh-release
+- name: staticfile-buildpack
+  url: https://github.com/cloudfoundry/staticfile-buildpack-release
+- name: go-buildpack
+  url: https://github.com/cloudfoundry/go-buildpack-release
+- name: ruby-buildpack
+  url: https://github.com/cloudfoundry/ruby-buildpack-release
+- name: python-buildpack
+  url: https://github.com/cloudfoundry/python-buildpack-release
+- name: php-buildpack
+  url: https://github.com/cloudfoundry/php-buildpack-release
+- name: nginx-buildpack
+  url: https://github.com/cloudfoundry/nginx-buildpack-release
+- name: nodejs-buildpack
+  url: https://github.com/cloudfoundry/nodejs-buildpack-release
+- name: binary-buildpack
+  url: https://github.com/cloudfoundry/binary-buildpack-release
+- name: java-offline-buildpack
+  url: https://github.com/cloudfoundry/java-offline-buildpack-release
+- name: java-buildpack
+  url: https://github.com/cloudfoundry/java-buildpack-release
+- name: dotnet-core-buildpack
+  url: https://github.com/cloudfoundry/dotnet-core-buildpack-release
+- name: r-buildpack
+  url: https://github.com/cloudfoundry/r-buildpack-release
+- name: hwc-buildpack
+  url: https://github.com/cloudfoundry-incubator/hwc-buildpack-release
+- name: concourse
+  url: https://github.com/concourse/concourse-bosh-release
+- name: turbulence
+  url: https://github.com/cppforlife/turbulence-release
+- name: jmeter-tornado
+  url: https://github.com/jamlo/jmeter-bosh-release
+- name: bosh-lemon-release
+  url: https://github.com/pivotal-cf-experimental/bosh-lemon-release
+- name: openvpn
+  url: https://github.com/dpb587/openvpn-bosh-release
+- name: jumpbox
+  url: https://github.com/cloudfoundry-community/jumpbox-boshrelease
+- name: kafka-service-broker
+  url: https://github.com/cloudfoundry-community/kafka-service-broker-boshrelease
+- name: bosh-aws-tools
+  url: https://github.com/cloudfoundry/bosh-aws-tools-release
+- name: cf-riak-cs
+  url: https://github.com/cloudfoundry/cf-riak-cs-release
+- name: garden-linux
+  url: https://github.com/cloudfoundry-incubator/garden-linux-release
+- name: docker
+  url: https://github.com/cloudfoundry-incubator/docker-boshrelease
+- name: mesos
+  url: https://github.com/cf-platform-eng/mesos-boshrelease
+- name: shipyard
+  url: https://github.com/cf-platform-eng/shipyard-boshrelease
+- name: ""
```

Comment:
Cut-and-paste leftover?

Reply:
I agree it is weird that we are testing for a non-empty string. It should be ReleaseSourceTypeBOSHIO whenever NewBOSHIOReleaseSource is called.